Tasks/DotNetCoreInstallerV1/task.json

```diff
@@ -0,0 +1,68 @@
+{
+    "id": "B0CE7256-7898-45D3-9CB5-176B752BFEA6",
+    "name": "DotNetCoreInstaller",
+    "friendlyName": ".NET Core Tool Installer",
+    "description": "Acquires a specific version of .NET Core from internet or the tools cache and adds it to the PATH. Use this task to change the version of .NET Core used in subsequent tasks.",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?linkid=853651)",
+    "category": "Tool",
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
```

Comment:
New major versions need to offer releaseNotes.  see https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/PowerShellV2/task.json#L22

When the UI advertises a new major version it suggests they should try it because it offers xxx (string from releaseNotes).